### DISCUSSION TITLE
Archivments

### DISCUSSION THREAD

**Discussion by ChewyToast** [community/community#34701] [Other Feature Feedback, Questions, & Ideas] [answered]
Could someone tell me the archivments that can be obtained on github, is there a list?

**Comment by Anmol-Baranwal** [comment, marked as answer]
I think you want to know about achievement badges on GitHub. Right?

First of all, there are different badges and each one has its unique way =>

You can see the list of badges [here](https://github.com/drknzz/GitHub-Achievements) or [here](https://github.com/Schweinepriester/github-profile-achievements)

List of Badges -
- Starstruck
- Quickdraw
- Pull Shark
- Pair Extraordinaire &nbsp;&nbsp;&nbsp;&nbsp; Visit [here](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) for more detail
- Galaxy Brain
- YOLO
- Public Sponsor

If you have any doubt regarding this, feel free to ask.

**Note**: Don't try to test the things like Pair Extraordinaire in the private testing repository. It will count and will be inaccessible if you delete the repository. Since it is due to the private repository, it will not have much weight.

**Reply by Anmol-Baranwal** [reply]
@ChewyToast please mark this as an answer if you have no further queries so that others can know that the question has been answered.

**Reply by ChewyToast** [reply]
Thanks for the info man! Have a good day

**Comment by masonzmz** [comment]
https://github.com/Schweinepriester/github-profile-achievements

**Comment by Ankit-Sharma1011** [comment]
Yes! GitHub has Achievements that you can earn through different activities, such as getting pull requests merged, having discussion answers accepted, co-authoring PRs, getting stars on a repository, etc.

Some of the achievements include:

🦈 Pull Shark
🧠 Galaxy Brain
🤝 Pair Extraordinaire
⚡ Quickdraw
⭐ Starstruck
🛸 YOLO

GitHub doesn't currently have a complete official list of all achievements, but there are community-maintained lists that track them. You can also see the achievements you've earned directly on your GitHub profile.
